setup.cfg

```diff
@@ -26,4 +26,6 @@ ignore =
     E741
     # E722: do not use bare except'
     E722
+    # W504: line break after binary operator
+    W504
```

Comment:
do we really want this? Seems like a code style thing we would not want to be just an after-though in some PR.

Reply:
My flake8 doesn't complain about it, but without that the CI complains for code I didn't touch in this PR :(